plugins/webgui/public/translate/en-US.js

```diff
@@ -44,18 +44,18 @@ module.exports = {
   '流量：': 'Data Usage:',
   '周期：': 'Duration:',
   '到期时间：': 'Due Date:',
-  '最近连接：': 'Recent Connection:',
+  '最近连接：': 'Last Connected:',
```

Comment:
In Telegram we mean the user `was last seen`, and here we mean the account `was last connected`.